resources/assetstore/charts/minio/templates/secrets.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  accesskey: {{ .Values.accessKey | b64enc }}
-  secretkey: {{ .Values.secretKey | b64enc }}
+  accesskey: {{ if .Values.accessKey }}{{ .Values.accessKey | b64enc }}{{ else }}{{ randAlphaNum 40 | b64enc | quote }}{{ end }}
```

Comment:
Sorry, I missed that. Access Key can have at max 20 characters:
https://github.com/helm/charts/tree/master/stable/minio